### ISSUE TITLE
Fails to decompress certain types of archives

### ISSUE THREAD

**Issue by irori** [use-strict/7z-wasm#1]
```
$ wget https://www.7-zip.org/a/7z1900-extra.7z
$ npx 7z-wasm x 7z1900-extra.7z
```

This fails with the following error message:

```
sigaction: signal type not supported: this is a no-op.
sigaction: signal type not supported: this is a no-op.

7-Zip (z) 21.03 beta (LE) : Copyright (c) 1999-2021 Igor Pavlov : 2021-07-20
 32-bit ILP32 locale=C.UTF-8 Threads:1

Scanning the drive for archives:
1 file, 929117 bytes (908 KiB)

Extracting archive: 7z1900-extra.7z
--
Path = 7z1900-extra.7z
Type = 7z
Physical Size = 929117
Headers Size = 532
Method = LZMA:22 BCJ2
Solid = +
Blocks = 2


ERROR: errno=6 : Resource temporarily unavailable

Archives with Errors: 1

sigaction: signal type not supported: this is a no-op.
sigaction: signal type not supported: this is a no-op.
```

**Comment by irori** [comment]
The error was generated in `CMixerMT::Code()` (in CodeMixer2.cpp) when trying to create a worker thread by calling `CVirtThread::Create()`.

Maybe 7zz.wasm should be compiled with `-D_7ZIP_ST` (the single-thread compile flag).

**Comment by use-strict** [comment]
Fixed in v1.0.0-beta.3
